docker/build.dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.7-stretch
```

Comment:
This is the base build dockerfile. It is the base of the dev container and it can be used in an CI server if you want to add CI to this project.